go.sum

```diff
@@ -4,6 +4,7 @@ github.com/armon/consul-api v0.0.0-20180202201655-eb2c6b5be1b6/go.mod h1:grANhF5
 github.com/coreos/etcd v3.3.10+incompatible/go.mod h1:uF7uidLiAD3TWHmW31ZFd/JWoc32PjwdhPthX9715RE=
 github.com/coreos/go-etcd v2.0.0+incompatible/go.mod h1:Jez6KQU2B/sWsbdaef3ED8NzMklzPG4d5KIOhIy30Tk=
 github.com/coreos/go-semver v0.2.0/go.mod h1:nnelYz7RCh+5ahJtPPxZlU+153eP4D4r3EedlOD2RNk=
+github.com/cpiwarski/locksmith v0.0.0-20200309230844-ca59c5e0b595 h1:g166rP6EDdgRDDa3epip6Kvv9/oR+EKaC+aR7AkJzVM=
```

Comment:
You're importing a downstream fork here.  Not sure how this happened, but you should revert your edits to this file.